src/main/scala/com/typesafe/sbt/bundle/SbtBundle.scala

```diff
@@ -151,8 +163,22 @@ object SbtBundle extends AutoPlugin {
         stageBundle(bundleType.value)
       }.value
     }.value,
+    NativePackagerKeys.dist in BundleConfiguration := Def.taskDyn {
+      Def.task {
+        createConfiguration()
+      }.value
+    }.value,
+    NativePackagerKeys.stage in BundleConfiguration := Def.taskDyn {
+      Def.task {
+        stageConfiguration
+      }.value
+    }.value,
     NativePackagerKeys.stagingDirectory in Bundle := (target in Bundle).value / "stage",
-    target in Bundle := target.value / "bundle"
+    NativePackagerKeys.stagingDirectory in BundleConfiguration := (target in BundleConfiguration).value / "stage",
+    target in Bundle := target.value / "bundle",
+    target in BundleConfiguration := target.value / "bundle-configuration",
+    sourceDirectory in BundleConfiguration := sourceDirectory.value / "main" / "bundle-configuration",
```

Comment:
You want to drop the `main` here. There's no need to distinguish between main and test. Also, if you do this then it will work for Play's directory structure also.